src/main/java/org/folio/rest/support/EffectiveCallNumberComponentsUtil.java

```diff
@@ -2,17 +2,16 @@
 
 import static org.apache.commons.lang3.StringUtils.isNotBlank;
 
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 import org.apache.commons.lang3.StringUtils;
 import org.folio.rest.jaxrs.model.EffectiveCallNumberComponents;
 import org.folio.rest.jaxrs.model.HoldingsRecord;
 import org.folio.rest.jaxrs.model.Item;
 import org.folio.services.CallNumberUtils;
 
-import java.util.Objects;
-import java.util.Optional;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
-
```

Comment:
This module typically uses [Eclipse import order|https://issues.folio.org/browse/FOLIO-1667]. Please configure your IDE to use that import ordering to minimise the amount of unnecessary changes to import statements during pull requests.

Reply:
Yes, it looks I have eclipse formatter plugin turned off. I followed by link and only found import configs there attached ("Eclipse style imports.xml" and "folio.importorder " files). Could you please point where the "eclipse formatter config" could be taken?

Reply:
@oleksandr-yatsenko1 

> Could you please point where the "eclipse formatter config" could be taken?

I don't know what Eclipse formatter is. If it is an intelliJ plugin, I don't use it and cannot provide configuration. I believe some of the EPAM developers do use that plugin, maybe they can help you.

Reply:
Why I asked about this plugin, because it looks like the picture attached in the mentioned ticket is the screenshot from Eclipse Code Formatter plugin itself - https://issues.folio.org/secure/attachment/15917/ecf.png. And there are both configuration files specified one for importing settings and another one for formatting.

Reply:
I have found this https://wiki.folio.org/display/FOLIJET/Java+code+formatter+for+IDEs, it looks like it contains formatting config used by FOLIO's developers.

Reply:
I think that some of the EPAM developers use that plugin. This is not an agreed FOLIO method. 

I do not use it. I use the configuration settings within IntelliJ itself.

I suggest asking other EPAM folks who use IntelliJ for guidance on the plugin.

Reply:
I'm using checkstyle Google in code that I maintain. Then these discussions become mood. Not because that style is better than any other as such, but because this will be fixed at people desktop or in Jenkins.

Reply:
@adamdickmeiss 

> I'm using checkstyle Google in code that I maintain. Then these discussions become mood.

I agree that adopting a code style would be very helpful with these kinds of topics. I do regret not adopting them much earlier in the lifetime of these modules. 

At the moment, it is a matter of finding the time to implement the checks and make the existing code compliant. The Core Functional team has no capacity for such work.

As far as I am aware, FOLIO has no agreed standard for this.

Reply:
I have configured Eclipse Code Formatter plugin (because as I remember it was recommended: https://wiki.folio.org/display/FOLIJET/Java+code+formatter+for+IDEs) with google style configuration got from here https://github.com/google/styleguide/blob/gh-pages/eclipse-java-google-style.xml. And for imports arranging I use folio.importorder from the ticket https://issues.folio.org/browse/FOLIO-1667.

![image](https://user-images.githubusercontent.com/63074477/111458609-4ae39a00-8722-11eb-8a67-6a57a579e065.png)